src/Sylius/Bundle/InstallerBundle/Command/InstallCommand.php

```diff
@@ -1,139 +1,69 @@
-        }
-
+        $output->writeln('<comment>Step 2 of 4.</comment> <info>Setting up the database.</info>');
+        $this->commandExecutor->runCommand('sylius:install:database', array(), $output);
         $output->writeln('');
-        $output->writeln('<info>Administration setup.</info>');
-
-        $this->setupAdmin($output);
 
+        $output->writeln('<comment>Step 3 of 4.</comment> <info>Shop configuration.</info>');
+        $this->commandExecutor->runCommand('sylius:install:setup', array(), $output);
         $output->writeln('');
 
-        return $this;
-    }
-
-    protected function setupDatabase(InputInterface $input, OutputInterface $output)
-    {
-        $this
-            ->runCommand('doctrine:database:create', $input, $output)
-            ->runCommand('doctrine:schema:create', $input, $output)
-            ->runCommand('cache:clear', $input, $output)
-            ->runCommand('doctrine:phpcr:repository:init', $input, $output)
-            ->runCommand('sylius:search:index', $input, $output)
-            ->runCommand('assets:install', $input, $output)
-            ->runCommand('assetic:dump', $input, $output)
-        ;
-    }
-
-    protected function setupFixtures(InputInterface $input, OutputInterface $output)
-    {
-        $doctrineConfig = $this->getContainer()->get('doctrine.orm.entity_manager')->getConnection()->getConfiguration();
-        $logger = $doctrineConfig->getSQLLogger();
-        $doctrineConfig->setSQLLogger(null);
-        $this
-            ->runCommand('doctrine:fixtures:load', $input, $output)
-            ->runCommand('doctrine:phpcr:fixtures:load', $input, $output)
-        ;
-        $doctrineConfig->setSQLLogger($logger);
-    }
-
-    protected function setupAdmin(OutputInterface $output)
-    {
-        $dialog = $this->getHelperSet()->get('dialog');
-
-        $userClass = $this->getContainer()->getParameter('sylius.model.user.class');
-        $user = new $userClass;
-
-        $user->setUsername($dialog->ask($output, '<question>Username:</question>'));
-        $user->setPlainPassword($dialog->ask($output, '<question>Password:</question>'));
-        $user->setEmail($dialog->ask($output, '<question>Email:</question>'));
-        $user->setEnabled(true);
-        $user->setRoles(array('ROLE_SYLIUS_ADMIN'));
-
-        $em = $this->getContainer()->get('doctrine.orm.entity_manager');
-        $em->persist($user);
-        $em->flush();
-    }
+        $output->writeln('<comment>Step 4 of 4.</comment> <info>Installing assets.</info>');
+        $this->commandExecutor->runCommand('sylius:install:assets', array(), $output);
+        $output->writeln('');
 
-    protected function runCommand($command, InputInterface $input, OutputInterface $output)
-    {
-        $this
-            ->getApplication()
-            ->find($command)
-            ->run($input, $output)
-        ;
+        $map = array(
+            Kernel::ENV_DEV => '/app_dev.php',
```

Comment:
Installer is only for Core, so I'm not sure what is the problem here. The dependency was missing. We do not want a generic installer, cause it is not really easy to do and maintain.